src/components/custom/_signup-affiliate.js

```diff
@@ -0,0 +1,217 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { navigate } from 'gatsby'
+import AgreementLabel from './_agreement-label'
+import { Input, Button } from 'components/form'
+import { Header, Text, LinkText } from 'components/elements'
+import { localize } from 'components/localization'
+import device from 'themes/device.js'
+
+const SignupContent = styled.div`
+    width: 48.4rem;
+    display: flex;
+    justify-content: flex-start;
+    flex-direction: column;
+    padding: 4rem;
+    background-color: var(--color-white);
+    border-radius: 0.6rem;
+    box-shadow: 0 1.6rem 2rem 0 rgba(0, 0, 0, 0.1);
+
+    @media ${device.tablet} {
+        width: 53rem;
+    }
+    @media ${device.mobileL} {
+        width: 100%;
+        padding: 6rem 2rem;
+    }
+`
+const StyledHeader = styled(Header)`
+    @media ${device.tabletL} {
+        font-size: 3rem;
+        margin-bottom: 3rem;
+    }
+`
+
+const SubTitle = styled(Text)`
+    @media ${device.tabletL} {
+        font-size: 2rem;
+        margin-bottom: 1rem;
+    }
+`
+const StyledText = styled(Text)`
+    @media ${(props) => device.tabletL && props.notedBox} {
+        width: 13rem;
+    }
+    @media (max-width: 340px) {
+        width: 17rem;
+    }
+    @media ${device.tabletL} {
+        font-size: ${(props) => props.tabletFontSize || 'var(--text-size-xxs)'};
+    }
+`
+const NoteBox = styled.div`
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    min-height: 1.7rem;
+    padding: 8px 14px;
+    margin-top: 1.6rem;
+    border-radius: 0.4rem;
+    background-color: rgba(242, 243, 244, 0.56);
+
+    @media ${device.mobileL} {
+        min-height: auto;
+        padding: 13px 16px;
+    }
+`
+const InputGroup = styled.div`
+    position: relative;
+    width: 100%;
+    margin: 2.5rem 0 0.6rem;
+
+    @media ${device.mobileL} {
+        margin: 25px 0 16px 0;
+    }
+`
+const EmailButton = styled(Button)`
+    width: 100%;
+    font-size: 1.4rem;
+    margin-bottom: 0.4rem;
+    margin-top: 3.2rem;
+
+    @media ${device.tabletL} {
+        margin-top: 24px;
+    }
+
+    @media ${device.mobileL} {
+        font-size: 1.75rem;
+    }
```

Comment:
For anything that is `tabletL` and below, we normally use `px` instead of `rem` as our HTML font sizes change with respect to the screen with. You can refer to `global-style.js` for more info.